### PR TITLE
(from rancher/rancher) #40358 Include instructions for manually updating AlertmanagerConfig after chart update

### PR DESCRIPTION
**Pull request by martyav**
Addresses a request to document changes in the v2.6.12 release.

https://github.com/rancher/rancher/issues/40358

This needs to be merged only once v2.6.12 is out.

### CODE REVIEW

File: versioned_docs/version-2.6/reference-guides/monitoring-v2-configuration/receivers.md [hunk 1]
```diff
@@ -237,6 +237,18 @@ It is also possible to send alerts to multiple notification systems. One way is
 
 You can also set up multiple receivers by using the `continue` option for a route, so that the alerts sent to a receiver continue being evaluated in the next level of the routing tree, which could contain another receiver.
 
+## Configuring AlertmanagerConfig After A Chart Upgrade
+
+If you use SMS or Microsoft Teams alerts, and upgrade the Monitoring chart from 100.1.3+up19.0.3 to 100.2.0+up40.1.2, you may no longer receive alerts. This is due to a change that helps prevent naming collisions.
+
+You must manually edit the configuration map to continue to receive alerts:
+
+1. Click on **☰** and select **local**.
```
Comment (btat) [inline versioned_docs/version-2.6/reference-guides/monitoring-v2-configuration/receivers.md:246]: This should just be any cluster that has SMS/MS Teams alerts using the affected chart version and not specific to the local cluster (the description of the linked issue mentions downstream).

File: versioned_docs/version-2.6/reference-guides/monitoring-v2-configuration/receivers.md [hunk 2]
```diff
@@ -237,6 +237,18 @@ It is also possible to send alerts to multiple notification systems. One way is
...
+1. Select **More Resources** > **Core** > **ConfigMaps**.
+1. Find the row corresponding to the ConfigMap for **Sachet** (for SMS alerts) or **prom2teams** (for Microsoft Teams alerts). Click the **⋮** and select **Edit YAML**.
+1. Update the delimiters in the receiver's `name` field, replacing */* characters with *-*. For example, if the receiver's name is `Test-Alert`, update it to `Test/Alert`.
```
Comment (btat) [inline versioned_docs/version-2.6/reference-guides/monitoring-v2-configuration/receivers.md:249]:
```suggestion
1. Update the delimiters in the receiver's `name` field, replacing `/` characters with `-`. For example, if the receiver's name is `Test-Alert`, update it to `Test/Alert`.
```

**Review by btat — CHANGES_REQUESTED** [review]

File: versioned_docs/version-2.6/reference-guides/monitoring-v2-configuration/receivers.md (hunk 1 above)
Comment (geethub97) [inline versioned_docs/version-2.6/reference-guides/monitoring-v2-configuration/receivers.md:246]: This change applies for both local and downstream clusters.

File: versioned_docs/version-2.6/reference-guides/monitoring-v2-configuration/receivers.md [hunk 3]
```diff
@@ -237,6 +237,18 @@ It is also possible to send alerts to multiple notification systems. One way is
 
 You can also set up multiple receivers by using the `continue` option for a route, so that the alerts sent to a receiver continue being evaluated in the next level of the routing tree, which could contain another receiver.
 
+## Configuring AlertmanagerConfig After A Chart Upgrade
+
+If you use SMS or Microsoft Teams alerts, and upgrade the Monitoring chart from 100.1.3+up19.0.3 to 100.2.0+up40.1.2, you may no longer receive alerts. This is due to a change that helps prevent naming collisions.
```
Comment (prachidamle) [inline versioned_docs/version-2.6/reference-guides/monitoring-v2-configuration/receivers.md:242]: This is due to a change that was done in upstream https://github.com/prometheus-operator/prometheus-operator/ that helps prevent naming collisions.

File: versioned_docs/version-2.6/reference-guides/monitoring-v2-configuration/receivers.md (hunk 2 above)
Comment (geethub97) [inline versioned_docs/version-2.6/reference-guides/monitoring-v2-configuration/receivers.md:249]: This should be: replacing *-* characters with */*

This is still slightly confusing as you can have receivers with '-' following the \<namespace\>/\<alertManagerConfig\>/\<receiverName\> format: default/amc-7/prom2sms. 


Can we instead say: 
Update receiver's `name` field to the following format: `\<namespace\>/\<alertManagerConfig\>/\<receiverName\>`, replacing *-* delimiters with */*. For example, if the receiver's name is `default-amc-test-prom2sms`, update it to `default/amc-test/prom2sms`.

**Review by geethub97 — CHANGES_REQUESTED** [review]

**Review by btat — APPROVED** [review]

File: versioned_docs/version-2.6/reference-guides/monitoring-v2-configuration/receivers.md [hunk 4]
```diff
@@ -237,6 +237,18 @@ It is also possible to send alerts to multiple notification systems. One way is
 
 You can also set up multiple receivers by using the `continue` option for a route, so that the alerts sent to a receiver continue being evaluated in the next level of the routing tree, which could contain another receiver.
 
+## Configuring AlertmanagerConfig After A Chart Upgrade
+
+If you have a cluster that uses SMS or Microsoft Teams alerts, and upgrade the Monitoring chart from 100.1.3+up19.0.3 to 100.2.0+up40.1.2, you may no longer receive alerts. This is due to a change to [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator/) that helps prevent naming collisions.
```
Comment (martyav) [inline versioned_docs/version-2.6/reference-guides/monitoring-v2-configuration/receivers.md:242]:
```suggestion
If you have a cluster that uses SMS or Microsoft Teams alerts, and upgrade the Monitoring chart from 100.1.3+up19.0.3 to 100.2.0+up40.1.2, you may no longer receive alerts. This is due to a change to an upstream change to [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator/) that helps prevent naming collisions.
```

**Comment by martyav** [comment]
@geethub97 I implemented suggested changes in [7c18050](https://github.com/rancher/rancher-docs/pull/569/commits/7c18050bd166291f684720776d9f78117bf3ebfd). Once we have your approval we'll be able to merge. The v2.6.12 release notes contain a link to the updated page so we'll be able to merge those too

**Review by geethub97 — APPROVED** [review]

**Review by prachidamle — APPROVED** [review]

**Pull request merged**